diff -ur include.orig/bullet/BulletCollision/BroadphaseCollision/btDbvt.h include/bullet/BulletCollision/BroadphaseCollision/btDbvt.h
--- include.orig/bullet/BulletCollision/BroadphaseCollision/btDbvt.h	2013-08-02 12:05:47.142260841 -0700
+++ include/bullet/BulletCollision/BroadphaseCollision/btDbvt.h	2013-08-02 12:30:47.862210634 -0700
@@ -205,6 +205,7 @@
 	{
 		const btDbvtNode*	node;
 		int			mask;
+		sStkNP() {}
 		sStkNP(const btDbvtNode* n,unsigned m) : node(n),mask(m) {}
 	};
 	struct	sStkNPS
diff -ur include.orig/bullet/BulletDynamics/Vehicle/btWheelInfo.h include/bullet/BulletDynamics/Vehicle/btWheelInfo.h
--- include.orig/bullet/BulletDynamics/Vehicle/btWheelInfo.h	2013-08-02 12:05:47.174260840 -0700
+++ include/bullet/BulletDynamics/Vehicle/btWheelInfo.h	2013-08-02 12:16:59.262238355 -0700
@@ -105,6 +105,11 @@
 
 	}
 
+	// This constructor is only here as a workaround for
+	// btAlignedObjectArray.  It is not intended to be used under normal
+	// conditions:
+	btWheelInfo() {}
+
 	void	updateWheel(const btRigidBody& chassis,RaycastInfo& raycastInfo);
 
 	btScalar	m_clippedInvContactDotSuspension;
diff -ur include.orig/bullet/LinearMath/btHashMap.h include/bullet/LinearMath/btHashMap.h
--- include.orig/bullet/LinearMath/btHashMap.h	2013-08-02 12:05:47.146260840 -0700
+++ include/bullet/LinearMath/btHashMap.h	2013-08-02 12:33:34.682205053 -0700
@@ -30,7 +30,7 @@
 		return m_hash;
 	}
 
-	btHashString(const char* name)
+	btHashString(const char* name = "")
 		:m_string(name)
 	{
 		/* magic numbers from http://www.isthe.com/chongo/tech/comp/fnv/ */
@@ -79,7 +79,7 @@
 {
 	int	m_uid;
 public:
-	btHashInt(int uid)	:m_uid(uid)
+	btHashInt(int uid = 0)	:m_uid(uid)
 	{
 	}
 
@@ -120,7 +120,7 @@
 
 public:
 
-	btHashPtr(const void* ptr)
+	btHashPtr(const void* ptr = NULL)
 		:m_pointer(ptr)
 	{
 	}
